src/users/users.service.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  ConflictException,
+  Injectable,
+  InternalServerErrorException,
+} from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { UpdateUserDetailsDto } from 'src/auth/dto/updateUser-userDetails.dto';
+import { User } from 'src/auth/user.entity';
+import { UsersRepository } from 'src/auth/users.respository';
+import { UserDetailsRepository } from 'src/user-details/user-details.repository';
+import * as fs from 'fs';
+
+@Injectable()
+export class UsersService {
+  constructor(
+    @InjectRepository(UsersRepository)
+    private userRepository: UsersRepository,
+    @InjectRepository(UserDetailsRepository)
+    private userDetailsRepository: UserDetailsRepository,
+  ) {}
+
+  async getProfileImage(user: User, res): Promise<Object> {
```

Comment:
Should return <any> type
![return-file](https://user-images.githubusercontent.com/77193627/156460811-d5211b26-c7b3-4a24-882f-988a3223fece.png)
.